test/testFunctions.js

```diff
@@ -28,34 +49,68 @@ const getContracts = async () => {
   return deployedContracts
 }
 
+/**
+ * getBalance of Acct and Tokens
+ * @param {address} acct
+ * @param {address} token
+ */
+const getBalance = async (acct, token) => {
+  const { DutchExchange: dx } = await getContracts()
+  return (await dx.balances.call(token.address, acct)).toNumber()
+}
+
 /**
  * >setupTest()
- * @param {Array[address]} accounts => ganache-cli accounts passed in globally
- * @param {Object}         contract => Contract object obtained via: const contract = await getContracts() (see above)
+ * @param {Array[address]} accounts         => ganache-cli accounts passed in globally
+ * @param {Object}         contract         => Contract object obtained via: const contract = await getContracts() (see above)
+ * @param {Object}         number Amounts   => { ethAmount = amt to deposit and approve, gnoAmount = for gno, ethUSDPrice = eth price in USD }
  */
-const setupTest = async (accounts, {
-  DutchExchange: dx, EtherToken: eth, TokenGNO: gno, PriceOracle: oracle,
-}) => {
+const setupTest = async (
+  accounts,
+  {
+    DutchExchange: dx,
+    EtherToken: eth,
+    TokenGNO: gno,
+    PriceOracle: oracle,
+  },
+  {
+    startingETH = 50..toWei(),
+    startingGNO = 200..toWei(),
+    ethUSDPrice = 60000,
+  }) => {
   // Await ALL Promises for each account setup
   await Promise.all(accounts.map((acct) => {
     /* eslint array-callback-return:0 */
     if (acct === accounts[0]) return
 
-    eth.deposit({ from: acct, value: 10 ** 9 })
-    eth.approve(dx.address, 10 ** 9, { from: acct })
-    gno.transfer(acct, 10 ** 18, { from: accounts[0] })
-    gno.approve(dx.address, 10 ** 18, { from: acct })
+    eth.deposit({ from: acct, value: startingETH })
+    eth.approve(dx.address, startingETH, { from: acct })
+    gno.transfer(acct, startingGNO, { from: accounts[0] })
+    gno.approve(dx.address, startingGNO, { from: acct })
   }))
   // Deposit depends on ABOVE finishing first... so run here
   await Promise.all(accounts.map((acct) => {
     if (acct === accounts[0]) return
 
-    dx.deposit(eth.address, 10 ** 9, { from: acct })
-    dx.deposit(gno.address, 10 ** 18, { from: acct })
+    dx.deposit(eth.address, startingETH, { from: acct })
+    dx.deposit(gno.address, startingGNO, { from: acct })
   }))
   // add token Pair
   // updating the oracle Price. Needs to be changed later to another mechanism
-  await oracle.updateETHUSDPrice(60000, { from: accounts[0] })
+  await oracle.updateETHUSDPrice(ethUSDPrice, { from: accounts[0] })
+
+  const gnoAcctBalances = await Promise.all(accounts.map(accts => getBalance(accts, gno)))
+  const ethAcctBalances = await Promise.all(accounts.map(accts => getBalance(accts, eth)))
+
+  gnoAcctBalances.forEach((bal, i) => {
+    if (i === 0) return
+    assert.equal(bal, startingGNO)
+  })
```

Comment:
An alternative code would be:
```
gnoAcctBalances.slice(1).forEach(bal => assert.equal(bal, startingGNO))
```
but whichever you prefer.

Reply:
@Velenir way cleaner. Done